### PR TITLE
Adding vbms

### PR DESCRIPTION
**Pull request by shanear**

### CODE REVIEW

**Comment by orischwartz** [comment]
I reviewed the JRuby changes, can't set up VBMS env quite yet. @paultag to approve that

File: config/secrets.yml [hunk 1]
```diff
@@ -13,13 +13,14 @@
...
   vbms:
-    url: 'url'
-    env_dir: 'env_dir'
-    keyfile: 'keyfile'
-    saml: 'saml'
-    keypass: 'keypass'
-    cacert: 'cacert'
-    cert: 'cert'
+    url: https://filenet.uat.vbms.aide.oit.va.gov/vbmsp2-cms/streaming/eDocumentService-v4
+    env_dir: /Users/vacorusses3/Developer/deployment/decrypted_creds/uat_creds
```
Comment (paultag) [inline config/secrets.yml]: Hardcoded here; can we use a relative path?

File: vendor/gems/connect_vbms/src/vbms/client.rb [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+      @http_client = http_client
+    end
+
+    def log(event, data)
+      @logger.log(event, data) if @logger
+    end
+
+    # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
+    def send_request(request)
+      puts "SEND REQUEST VBMS"
```
Comment (paultag) [inline vendor/gems/connect_vbms/src/vbms/client.rb]: This appeas to be a local modification to the vendored library. We shouldn't modify stuff we vendor, it makes it very hard to back stuff off (also, this looks like it was just a debugging `puts`)

File: vendor/gems/connect_vbms/src/vbms/client.rb [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+        @keypass,
+        request.name)
+      doc = Nokogiri::XML(output)
+      inject_saml(doc)
+      remove_must_understand(doc)
+
+      body = create_body(request, doc)
+
+      puts "CONTACTING VBMS"
+      puts @endpoint_url
```
Comment (paultag) [inline vendor/gems/connect_vbms/src/vbms/client.rb]: This appeas to be another local modification

**Comment by shanear** [comment]
thanks @paultag, good notes :us:. Made the requested changes.

**Comment by paultag** [comment]
@shanear awesome, looks great now. Thanks!

**Pull request merged**